src/app/globals.css

```
@@ -97,4 +100,4 @@
   box-shadow: -1px 1px 30px -13px rgba(255, 255, 255, 1);
 }
 
-@import '@fontsource/inter';
+@import '@fontsource/inter';
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)